tests/test_translation/test_g2g_serialization.py

```diff
@@ -0,0 +1,618 @@
+    - term: ECO:0000314
+      reference: PMID:11751572
+    term: GO:0004708
+  occurs_in:
+    evidence:
+    - term: ECO:0000318
+      reference: PMID:21873635
+    term: GO:0005737
+  part_of:
+    evidence:
+    - term: ECO:0000315
+      reference: PMID:12142542
+    term: GO:0140367
+  causal_associations:
+  - evidence:
+    - term: ECO:0000315
+      reference: PMID:12142542
+    predicate: RO:0002629
+    downstream_activity: gomodel:568b0f9600000284/568b0f9600000285
+- id: gomodel:568b0f9600000284/568b0f9600000285
+  enabled_by:
+    term: WB:WBGene00004055
+  molecular_function:
+    evidence:
+    - term: ECO:0000314
+      reference: PMID:20369020
+    term: GO:0004707
+  occurs_in:
+    evidence:
+    - term: ECO:0000314
+      reference: PMID:20133945
+    term: GO:0005829
+  part_of:
+    evidence:
+    - term: ECO:0000315
+      reference: PMID:12142542
+    term: GO:0140367
+  causal_associations:
+  - evidence:
+    - term: ECO:0000314
+      reference: PMID:20369020
+    predicate: RO:0002629
+    downstream_activity: gomodel:568b0f9600000284/568b0f9600000287
+- id: gomodel:568b0f9600000284/568b0f9600000287
+  enabled_by:
+    term: WB:WBGene00000223
+  molecular_function:
+    evidence:
+    - term: ECO:0000250
+      reference: PMID:20369020
+    term: GO:0000981
+  occurs_in:
+    evidence:
+    - term: ECO:0000314
+      reference: PMID:20369020
+    term: GO:0005634
+  part_of:
+    evidence:
+    - term: ECO:0000315
+      reference: PMID:20369020
+    term: GO:0140367
+objects:
+- id: WB:WBGene00006575
+  label: tir-1 Cele
+- id: WB:WBGene00006599
+  label: tpa-1 Cele
+- id: WB:WBGene00012019
+  label: dkf-2 Cele
+- id: WB:WBGene00003822
+  label: nsy-1 Cele
+- id: WB:WBGene00004758
+  label: sek-1 Cele
+- id: WB:WBGene00004055
+  label: pmk-1 Cele
+- id: WB:WBGene00000223
+  label: atf-7 Cele
+- id: GO:0035591
+  label: signaling adaptor activity
+- id: GO:0004674
+  label: protein serine/threonine kinase activity
+- id: GO:0004709
+  label: MAP kinase kinase kinase activity
+- id: GO:0004708
+  label: MAP kinase kinase activity
+- id: GO:0004707
+  label: MAP kinase activity
+- id: GO:0000981
+  label: DNA-binding transcription factor activity, RNA polymerase II-specific
+- id: GO:0005737
+  label: cytoplasm
+- id: GO:0009898
+  label: cytoplasmic side of plasma membrane
+- id: GO:0005829
+  label: cytosol
+- id: GO:0005634
+  label: nucleus
+- id: GO:0140367
+  label: antibacterial innate immune response
+- id: GO:0002225
+  label: positive regulation of antimicrobial peptide production
+- id: RO:0002629
+  label: directly positively regulates
+"""
+
+
+@pytest.fixture
+def readme_model():
+    """Create a GO-CAM model from the README example."""
+    gocam_data = yaml.safe_load(README_GOCAM_YAML)
+    return Model.model_validate(gocam_data)
+
+
+@pytest.fixture
+def translator():
+    """Create a ModelNetworkTranslator instance."""
+    return ModelNetworkTranslator()
+
+
+class TestG2GSerialization:
```

Comment:
Not sure why this is a class-based test?

For that matter I would expect that everything that tests the `ModelNetworkTranslator` class would be in a file called `tests/test_translation/test_model_network_translator.py`. Do the tests need to be split into two files for some reason?

Reply:
fixed!